examples/multiple_mzr_clusters/main.tf

```diff
@@ -174,20 +174,13 @@ data "ibm_container_cluster_config" "cluster_config_c2" {
 ########################################################################################################################
 
 module "observability_instances" {
-  source  = "terraform-ibm-modules/observability-instances/ibm"
-  version = "2.18.1"
-  providers = {
-    logdna.at = logdna.at
-    logdna.ld = logdna.ld
-  }
+  source                         = "terraform-ibm-modules/observability-instances/ibm"
+  version                        = "3.0.0"
   resource_group_id              = module.resource_group.resource_group_id
   region                         = var.region
-  log_analysis_plan              = "7-day"
   cloud_monitoring_plan          = "graduated-tier"
-  activity_tracker_provision     = false
   enable_platform_logs           = false
   enable_platform_metrics        = false
-  log_analysis_instance_name     = "${var.prefix}-logdna"
   cloud_monitoring_instance_name = "${var.prefix}-sysdig"
   cloud_logs_provision           = false
```

Comment:
This needs to be set to `true`.
and the [logs agents](https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/variables.tf#L233) should send logs to the Cloud logs.

Reply:
@Aashiq-J we dont need to test deploying log agents in base OCP module